src/screens/PMAuthenticationProcessor/PMAuthenticationTableEntity.res

```diff
@@ -0,0 +1,90 @@
+open ConnectorTypes
+
+type colType =
+  | Name
+  | TestMode
+  | Status
+  | Disabled
+
+  | ProfileId
+  | ProfileName
+  | ConnectorLabel
+
+let defaultColumns = [Name, ProfileId, ProfileName, ConnectorLabel, Status, Disabled, TestMode]
+
+let getHeading = colType => {
+  switch colType {
+  | Name => Table.makeHeaderInfo(~key="connector_name", ~title="Processor", ~showSort=false)
```

Comment:
Can we remove showSort as @sagarnaikjuspay was removing it

Reply:
that PR is not merged

Reply:
okay as sagar to remove from this place if this gets merged first